lib/handlerTemplate.js

```diff
@@ -192,7 +192,11 @@ function prepareRequestChain(conf) {
         if (requestConf.request) {
             template = new Template(requestConf.request);
             requestSpec.request = function(restbase, context) {
-                var req = template.eval(context);
+                var req = template.expand(context);
+                if (!requestConf.request.method) {
+                    // Do we really want this?
+                    req.method = context.request.method || 'get';
```

Comment:
Yeah, but it seems that quite a few of our specs currently rely on this behavior. Many POST end points (transforms, page save api) don't seem to specify a method in the x-request-handler.

I'm definitely +1 on keeping things simple & explicit, and only offering 'get' fall-back, but we should get agreement in the team before adjusting the specs.

@d00rman, @eevans: What's your take on this?